webpack.config.js

```diff
@@ -123,7 +127,12 @@ const config = {
 			memo[ name ] = `./${ path }`;
 			return memo;
 		}, {} ),
-		packageNames.reduce( ( memo, packageName ) => {
+		gutenbergPackages.reduce( ( memo, path ) => {
```

Comment:
Any reason to rename `packageName` to `path`? It's still just using the `packageName` and I like the explicitness of the variable name. Plus it will hopefully discourage someone from including more nested paths.

Reply:
I guess it's copy and paste thing. Will fix :)